test/integration/targets/ec2_instance/tasks/termination_protection.yml

```
@@ -24,62 +11,60 @@
         termination_protection: true
         instance_type: "{{ ec2_instance_type }}"
```

```suggestion
        instance_type: "{{ ec2_instance_type }}"
        state: running
```

The state here should really match the state when we actually create the instance